src/emm/services/modules_service.py

```diff
@@ -225,9 +249,119 @@ def git_commit_modules(self, commit: str) -> None:
 
         :param commit: Commit content for all changes.
         """
-        enabled_modules = self.get_all_modules(True)
         self.git_service.commit_all(commit)
+        enabled_modules = self.get_all_modules(True)
         for module in enabled_modules:
             module_data = self.get_module_data(module)
             module_location = Path(module_data.prefix) / module
             self.git_service.commit_all(commit, module_location)
+
+    def validate_github_authentication(self) -> bool:
+        """Check if github CLI already authenticated."""
+        return self.github_service.validate_github_authentication()
+
+    def add_pr_links(self, comments: Dict[str, str]) -> List[PullRequestInfo]:
```

Comment:
`add_pr_links` should probably be renamed, since it isn't really adding anything.